docker/README.md

```diff
@@ -0,0 +1,29 @@
+# Ed-Fi-Analytics-Middle-Tier docker local configuration
+
+This docker container satisfies the user story:
+> As an AMT developer, I want to run integration tests using PostgreSQL and SQL Server in Docker containers on my localhost.
+
+To build and run this container follow this steps:
+
+- run in terminal
+> docker-compose build
+- when container is created, run the following in terminal
+> docker-compose up
```

Comment:
`docker-compose up -d` is the normal recommendation. FYI, if you are running a recent version of Docker Desktop, you can remove the dash and just use `docker compose`.

This should also be surrounded by a code fence, like below. Unfortunately, I can't show this properly in GitHub because I can't figure out how to properly escape backticks! Thus I pasted in a screenshot.

![image](https://user-images.githubusercontent.com/9324390/148092244-ef11493c-758a-42d6-8f57-ff44070df273.png)